controls/def.cf

```diff
@@ -315,6 +333,10 @@ bundle common def
       ifvarclass => "$(augments_classes_data_keys)";
 
       "$(const.t) $(defvars) = $($(defvars))";
+      "DEBUG $(this.bundle): Agent parsed augments_file"
+        ifvarclass => "have_augments_file.feature_augments";
+      "DEBUG $(this.bundle): Policy parsed augments_file"
```

Comment:
Maybe add a report for the case where we do nothing because `feature_augments` is defined? Not sure...